src/app/components/header/header.component.ts

```diff
@@ -27,13 +27,24 @@ export class HeaderComponent implements OnInit {
   @Input() public currentUrl: string;
   @Output() public navigate = new EventEmitter<string>();
 
+  skipLinkNavigate: string = '';
+  contentHash: string = '#content';
```

Comment:
need public accessor declaring

Reply:
Will sort this guys @GaryScottHMCTS and @phillip-whitaker-hmcts